src/Medidata.ZipkinTracer.HttpModule/ZipkinConfig.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+
+namespace Medidata.ZipkinTracer.HttpModule
+{
+    public class ZipkinConfig : IZipkinConfig
```

Comment:
@kenyamat I suggest to have this implemented at Gambit or the client application side. Example is Get method can be getting settings key from environment variables